metadata.txt

```diff
@@ -35,7 +37,7 @@ homepage=http://www.inasafe.org/
 tracker=https://github.com/inasafe/inasafe/issues
 repository=https://github.com/inasafe/inasafe
 icon=resources/img/icons/icon.png
-# experimental flag
+# experimenal flag
```

Comment:
What are you doing here? ;-)
Can you update the version number at the same time ?